### ISSUE TITLE
Java 13 build failure

### ISSUE THREAD

**Issue by externl** [zeroc-ice/ice-demos#104]
```
❯ ./gradlew -PdevRepo=https://builds.testing.zeroc.com/nexus/repository/maven.ice-v3.7.3 build

FAILURE: Build failed with an exception.

* What went wrong:
Could not open settings remapped class cache for 8ly38xblk7bu1s2fvxwf4d9ej (/Users/joe/.gradle/caches/5.0/scripts-remapped/settings_7ib3gm83md6wem6vmq04qnvkb/8ly38xblk7bu1s2fvxwf4d9ej/settingsa9190c3ddc71e1450eb3f3e7bc6a334f).
> Could not open settings generic class cache for settings file '/Users/joe/src/github.com/zeroc-ice/ice-demos/java/settings.gradle' (/Users/joe/.gradle/caches/5.0/scripts/8ly38xblk7bu1s2fvxwf4d9ej/settings/settingsa9190c3ddc71e1450eb3f3e7bc6a334f).
   > BUG! exception in phase 'semantic analysis' in source unit '_BuildScript_' Unsupported class file major version 57

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 0s
```

**Comment by externl** [comment]
`java-combat` works fine, however it uses `gradle-4.10.2` whereas `java` is using `gradle-5.0-bin.zip`

**Comment by externl** [comment]
Probably related to https://github.com/gradle/gradle/issues/8681

**Comment by pepone** [comment]
I think this is already fixed with upgrade to gradle 6.3
